suite-native/accounts/src/components/AccountSelectBottomSheet.tsx

```diff
@@ -93,14 +90,16 @@ export const AccountSelectBottomSheet = React.memo(
         );
 
         return (
-            <BottomSheet isVisible isCloseDisplayed={false} onClose={onClose} isScrollable={false}>
-                <FlashList
-                    data={data}
-                    renderItem={renderItem}
-                    contentContainerStyle={applyStyle(contentContainerStyle)}
-                    estimatedItemSize={76}
-                />
-            </BottomSheet>
+            <BottomSheetFlashList<AccountSelectBottomSheetSection>
+                isVisible
+                isCloseDisplayed={false}
+                onClose={onClose}
+                data={data}
+                renderItem={renderItem}
+                estimatedItemSize={ESTIMATED_ITEM_SIZE}
+                estimatedListHeight={ESTIMATED_ITEM_SIZE * data.length}
```

Comment:
This order makes more sense to me. Not a big deal.
```suggestion
                estimatedListHeight={data.length * ESTIMATED_ITEM_SIZE}
```

Reply:
It doesn't make sense for me :D